kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -21,12 +22,13 @@ class KafkaCheck(AgentCheck):
         'ssl_password': {'name': 'tls_private_key_password'},
     }
 
-    def __init__(self, name, init_config, instances):
+    def __init__(self, name, init_config, instances, client=KafkaPythonClient):
         super(KafkaCheck, self).__init__(name, init_config, instances)
         self.config = KafkaConfig(self.init_config, self.instance)
         self._context_limit = self.config._context_limit
         tls_context = self.get_tls_context()
-        self.client = make_client(self.config, tls_context, self.log)
+        self.client = make_client(client, self.config, tls_context, self.log)
```

Comment:
I made the client initialization configurable so that it's easier to test each client.